### PR TITLE
Feature/export metrics

### PR DESCRIPTION
**Pull request by Nainterceptor**
Checklist

- [x] I've added a mention in CHANGELOG.md

### CODE REVIEW

**Comment by Nainterceptor** [comment]
[Metrics / Logs dashboard](https://trello.com/c/fWgY8UGA/297-metrics-logs-dashboard)

File: .helm-chart/templates/prometheus-service-monitor.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+metadata:
+  name: {{ $fullName }}
+  labels:
+    app: {{ $name }}
+    heritage: {{ .Release.Service }}
+    release: prometheus
+spec:
+  endpoints:
+    - interval: 30s
+      port: metrics
```
Comment (Scttpr) [inline .helm-chart/templates/prometheus-service-monitor.yaml]: Is it a variable ? Or you don't need to expose a port number ?

File: Dockerfile [hunk 2]
```diff
@@ -28,6 +28,7 @@ COPY . /usr/src/app/
 COPY ./config.json.dist ./config.json
 
 EXPOSE 1337
+EXPOSE 1338
```
Comment (Scttpr) [inline Dockerfile]: Oh ! I thought it overrided line 30 but I guess you expose multiple ports for replica set ?

File: middlewares/metrics.mjs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import Prometheus from 'prom-client';
+
+const httpRequestDurationMicroseconds = new Prometheus.Histogram({
+  name: 'http_request_duration_ms',
+  help: 'Duration of HTTP requests in ms',
+  labelNames: ['method', 'route', 'code'],
+  buckets: [0.10, 5, 15, 50, 100, 200, 300, 400, 500],
+});
+
+export default () => async (ctx, next) => {
```
Comment (Scttpr) [inline middlewares/metrics.mjs]: Here you export a function with no params and in app.mjs you give it `metricsInterval`

File: .helm-chart/templates/prometheus-service-monitor.yaml (hunk 1 above)
Comment (Nainterceptor) [reply .helm-chart/templates/prometheus-service-monitor.yaml]: It will use the name in services :)

File: Dockerfile (hunk 2 above)
Comment (Nainterceptor) [reply Dockerfile]: 1337 for the application ; 1338 for metrics

File: middlewares/metrics.mjs (hunk 3 above)
Comment (Nainterceptor) [reply middlewares/metrics.mjs]: You're right !

**Pull request merged**